internal/gen/schema.go

```diff
@@ -135,7 +135,11 @@ func (g *Generator) mkFieldSchema(fd protoreflect.FieldDescriptor, description s
 		return ogen.NewSchema().SetType("number").SetFormat("double").SetDeprecated(isDeprecated(fd.Options())).SetDescription(mkDescription(description)), nil
 
 	case protoreflect.StringKind:
-		return ogen.NewSchema().SetType("string").SetDeprecated(isDeprecated(fd.Options())).SetDescription(mkDescription(description)), nil
+		schema := ogen.NewSchema().SetType("string").SetDeprecated(isDeprecated(fd.Options())).SetDescription(mkDescription(description))
+		if isFieldUUID4Format(fd.Options()) {
```

Comment:
ogen supports `ipv4`/`ipv6`/`ip` formats, probably we should handle these too?

Reply:
Ok

Reply:
Done